eacl/types.go

```diff
@@ -83,11 +85,20 @@ func (u *ValidationUnit) WithSenderKey(v []byte) *ValidationUnit {
 	return u
 }
 
-// WithBearerToken configures ValidationUnit to use v as request's bearer token.
+// WithEACLTable configures ValidationUnit to use v as request's bearer token.
```

Comment:
non-functional change that does not relate commit, do it in a separate commit?

Reply:
Moved to an additional commit